src/index.js

```diff
@@ -35,15 +35,23 @@ const processArgs = args => {
   }
 }
 
+const makeCardName = (namespace, opts) => {
+    //TODO: Make this better
+    return namespace + "__" + opts.doc.split('\n')[0];
+}
+
 export default function(namespace = 'default') {
   const cards = []
   let nextId = 1
-  store.set(namespace, cards)
   return {
     card(...args) {
       const [content, opts] = processArgs(args)
       const CardImpl = typeof content === 'function' ? StatefulCard : Card
-      cards.push(<CardImpl {...opts} key={nextId++}>{content}</CardImpl>)
+      const card = <CardImpl {...opts} key={nextId++}>{content}</CardImpl>
+      const cardName = makeCardName(namespace, opts)
+      //TODO: Do we need to push the card to the cards list?
+      cards.push(card)
+      store.set(cardName, [card])
```

Comment:
While I assume this works, it also destroys the multiple cards per page setup in the process.

Ideally I'd like to see a change that enables 2 different views: one view that is more or less what we have today and one that shows each card on its own page.  That way we can have our cake and eat it too.

I think to do that you'll have to make changes on the rendering side, not on the data storage side (name changes so we can actually list all cards not withstanding)

Reply:
I just had another thought on this front.

What if we just make it so that each card is also a link to a page with just that single card?  So basically we keep the existing setup exactly as is, but on the namespace page, we make cards links to new pages that only show that single card.  In the end we'd have the index page which lists all namespaces, then on say the Timeline namespace page we'd still show all cards in the namespace but the Basic Timeline card would link to /Timeline/BasicTimeline which would be a page with just that single card.

The benefit of that approach would be that we'd be able to get to a single card per page view for the automation, but we'd also be able to show many cards per page (which is preferable for devs looking at changes).  We would also be able to add additional updates (such as multiple levels of namespacing for better organization) without changing the underlying setup.

Reply:
That's the way I think it should work, too

Reply:
yea, I really like the sound of approaching this from the display side.  (By the way, I'm not wed to anything here, it was a 15 minute attempt at seeing a change)

Reply:
I had a though the other day, the store is really just redux, and the view is just drawing the store in the format of the store, so maybe we could change the way the store works based on url or query parameter